src/firefly/js/core/background/JobInfo.jsx

```diff
@@ -130,6 +133,29 @@ function JobInfoDetails({jobId, ownerId, phase, creationTime, startTime, endTime
     );
 }
 
+function TAPDetails({jobInfo}) {
+    const lang = jobInfo?.parameters?.lang;
+    const params = jobInfo?.parameters || {};
+    const adql = params[Object.keys(params).find((k) => k.toLowerCase() === 'query')];
+
+    if (!adql || (lang && lang.toUpperCase() !== 'ADQL')) return null;
+
+    return (
+        <Stack spacing={0}>
+            <KeywordBlock label='ADQL QUERY'/>
+            <PrismADQLAware text={adql} sx={{marginBlock: '-8px'}}/>
```

Comment:
```suggestion
            <PrismADQLAware text={adql} sx={{marginBlock: '-8px', fontSize: 'fontSize.sm'}}/>
```
Everyone else using sm font size in this compact panel except this Prism block

Reply:
Did you mean fontSize: `var(--joy-fontSize-sm)`? Using `fontSize.sm` didn’t work for me.

Reply:
Oh, I got confused. The right syntax should be `fontSize: 'sm'` - sx should recognize it